app/views/shared/_user_menu.html.erb

```diff
@@ -0,0 +1,64 @@
+<div
+    class="w-full border-t border-gray-200/50 pt-6 pb-2 pr-2 bg-black flex flex-col gap-2"
+    data-controller="click-toggle"
+    data-click-toggle-flippable-class="hidden"
+    >
+
+    <a href="#"
+       data-action="click->click-toggle#toggleClasses:prevent"
+       class="flex justfiy-left gap-2 rounded-lg text-white text-sm hover:bg-[#28282b] p-2"
+    >
+      <%= icon "cog" , size: 20 %>
+      Settings
+    </a>
+
+    <a href="/logout" class="flex justify-left gap-2 hover:bg-[#28282b] rounded-lg text-white text-sm p-2">
+    <%= heroicon "arrow-right-start-on-rectangle" , options: { class: 'h-5 w-5 text-gray-300' } %>
```

Comment:
I think we should always use the wrapper method, <%= icon %>, rather than heroicon. It'll make future refactors easier. And it supports "class:" so you don't have to prefix with options:

Reply:
Agreed -- I didn't touch this code but I'll give it a once-over and bring it up to spec.